### PR TITLE
PORT-10619 Restructure Gitlab Integration Docs 

### PR DESCRIPTION
**Pull request by kodjomiles**
# Description

Changed the gitlab integration guide to use new integration guide structure 

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- Gitlab Into (/build-your-software-catalog/sync-data-to-catalog/git/gitlab)
- Gitlab Installation (/build-your-software-catalog/sync-data-to-catalog/git/gitlab/installation)

### CODE REVIEW

**Review by PeyGis — APPROVED** [review]
LGTM

File: docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/gitlab.md [hunk 1]
```diff
@@ -1,523 +1,67 @@
...
-    port:
-      entity:
-        mappings: ...
-```
-
-:::tip
-Pay attention to the value of the `blueprint` key, if you want to use a hardcoded string, you need to encapsulate it in 2 sets of quotes, for example use a pair of single-quotes (`'`) and then another pair of double-quotes (`"`)
-:::
-
-### Ingest files from your repositories
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/gitlab.md:162]: This is an important capability, it should not be removed. Please move it under the capabilities section with its own sub-header.

File: docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/installation.md [hunk 2]
```diff
@@ -16,20 +16,22 @@ This page will help you install Port's GitLab integration (powered by the Ocean
...
 This page outlines the following steps:
 
-- How to [create](#creating-a-gitlab-group-access-token) a GitLab group access token to give the integration permissions to query your GitLab account.
-- How to [configure](#configuring-the-gitlab-integration) and customize the integration before deploying it.
-- How to [deploy](#deploying-the-gitlab-integration) the integration in the configuration that fits your use case.
+- How to [create](#create-a-gitlab-group-access-token) a GitLab group access token to give the integration permissions to query your GitLab account.
+- How to [configure](#configure-the-gitlab-integration) and customize the integration before deploying it.
+- How to [deploy](#deploy-the-gitlab-integration) the integration in the configuration that fits your use case.
 
 :::note Prerequisites
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/installation.md:23]: Change to a header instead of an admonition.

**Review by hadar-co — CHANGES_REQUESTED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/gitlab.md (hunk 1 above)
Comment (kodjomiles) [reply docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/gitlab.md:162]: Just to confirm is it just the Ingest files from repo I should add or the other bit's that follows it are all to be added to capabilities, from where I see it, the data seems to be mapping configurations which has already been covered..
Is there something I am missing here ?

**Review by hadar-co — APPROVED** [review]

**Pull request merged**